collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt

```diff
@@ -49,13 +55,16 @@ class FormUriActivity : ComponentActivity() {
             logAnalytics(uriProjectId)
```

Comment:
Do you think we should move the analytics call so that it doesn't fire twice if the activity is recreated? Not a huge deal, but it does taint the numbers.

Reply:
Yeah why not. Done.